posthog/queries/trends/test/__snapshots__/test_person.ambr

```diff
@@ -17,7 +17,7 @@
      WHERE team_id = 2
        AND event = 'pageview'
        AND toDateTime(timestamp, 'UTC') >= toDateTime('2021-01-21 00:00:00', 'UTC')
-       AND toDateTime(timestamp, 'UTC') <= toDateTime('2021-01-21 23:59:59', 'UTC')
+       AND toDateTime(timestamp, 'UTC') <= toDateTime('2021-01-22 23:59:59', 'UTC')
```

Comment:
Flyby: this seems wrong, as now it's 2 days worth of people, instead of 1.

Reply:
Good catch, though it was actually because the test had filters set up in a way that the frontend would never actually send. Update those tests.